src/mudlet-lua/lua/generic-mapper/generic_mapper.xml

```diff
@@ -1468,13 +1470,13 @@ function map.setConfigs(key, val, sub_key)
                 if #long_dir &lt; #short_dir then long_dir, short_dir = short_dir, long_dir end
                 map.configs.lang_dirs[sub_key] = long_dir
                 map.configs.lang_dirs[short[sub_key]] = short_dir
-                map.echo(string.format("Direction %s, abbreviated as %s, now interpreted as %s.", long_dir, short_dir, sub_key))
+                map.echo(string.format("Direction/command %s, abbreviated as %s, now interpreted as %s.", long_dir, short_dir, sub_key))
```

Comment:
I don't know much about the script but are the directions used for more than movement commands? Otherwise, we could summarize all of them as `commands`, which is less awkward than the `/`.

Reply:
lang_dirs is used to send command to mud, and to detect exit from room description. Can be change to "Direction or command" if you don't like '/'

Reply:
It's not the `/` that irks me. But if it's not only input but also game output that uses the table, this is probably fine.